tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -496,9 +496,11 @@ void validate_single_case(CaseParam const& param) {
     execute_test(param, [&]() {
         ValidationCase const validation_case = create_validation_case(param);
         std::string const output_prefix = get_output_type(param.calculation_type, param.sym);
-        SingleData result = create_result_dataset(validation_case.input, output_prefix);
+        SingleData const result = create_result_dataset(validation_case.input, output_prefix);
 
         // create model and run
+        // TODO (mgovers): fix false positive of misc-const-correctness
+        // NOLINTNEXTLINE(misc-const-correctness,-warnings-as-errors)
```

Comment:
why do you need the `-warnings-as-errors` as well? shouldn't it just be

```suggestion
        // NOLINTNEXTLINE(misc-const-correctness)
```